src/app/ask-recovery/page.tsx

```diff
@@ -79,6 +81,20 @@ export default function AskRecovery() {
     setThreshold(value);
   };
 
+  const validateNewOwner = useCallback(
+    (address: string) => {
+      if (!isAddress(address))
+        return { isValid: false, reason: "Insert a valid address." };
+      if (address === safeAddress)
+        return {
+          isValid: false,
+          reason: "The safe address can't be an owner.",
```

Comment:
```suggestion
          reason: "This Safe address can't be an owner.",
```

should we add a better description of why that is?